drivers/input/input_ft5336.c

```diff
@@ -150,8 +152,23 @@ static int ft5336_init(const struct device *dev)
 
 	k_work_init(&data->work, ft5336_work_handler);
 
+	if (config->reset_gpio.port != NULL) {
+		/* Enable reset GPIO, and pull down */
+		r = gpio_pin_configure_dt(&config->reset_gpio, GPIO_OUTPUT_INACTIVE);
+		if (r < 0) {
+			LOG_ERR("Could not enable reset GPIO");
+			return r;
+		}
+		/* Delay 5 ms to reset controller */
+		k_sleep(K_MSEC(5));
```

Comment:
where are these 5ms documented?

Reply:
I've updated the comment based on the datasheet. It appears that the minimum reset time is 1ms, but I would rather keep the longer delay to be safe (as the reset time needs to be extended if the controller is just powering on)